wiring/inc/spark_wiring_system.h

```diff
@@ -261,17 +261,21 @@ class SystemClass {
 		set_flag(flag, false);
     }
 
+    inline bool enabled(system_flag_t flag) const {
+        return get_flag(flag) != 0;
+    }
+
 
 private:
 
-    inline uint8_t get_flag(system_flag_t flag)
+    static inline uint8_t get_flag(system_flag_t flag)
     {
         uint8_t value = 0;
         system_get_flag(flag, &value, nullptr);
         return value;
     }
 
-    inline void set_flag(system_flag_t flag, uint8_t value)
+    static inline void set_flag(system_flag_t flag, uint8_t value)
```

Comment:
The reason is that I added `System::enabled()` method which is `const` and thus requires `get_flag()` to be either static or `const` as well. Note that both `get_flag()` and `set_flag()` methods are in private section of the class, so there shouldn't be any compatibility problem if we change internal implementation later.